assets/scss/modules/_tooltip.scss

```diff
@@ -5,31 +5,29 @@
 /// @author gbaamonde@makingsense.com
 /// Classes for the tooltip component
 
-@mixin dp-tooltip($bg-tooltip, $font-size-tooltip, $font-color-tooltip) {
+@mixin dp-tooltip($font-size-tooltip, $font-color-tooltip) {
   position: relative;
   display: inline-block;
-  padding: 3px $dp-spaces--lv1;
-  margin: 0;
-  background: $bg-tooltip;
-  border-radius: $border-radius;
-  font-size: $font-size-tooltip;
-  text-transform: uppercase;
-  color: $font-color-tooltip;
+  padding: 3px $dp-spaces--lv0;
+  margin: $dp-spaces--lv0;
+  color: $dp-color-darkyellow;
+  cursor: pointer;
 
   .tooltiptext {
     visibility: hidden;
     position: absolute;
-    width: auto;
     min-width: 170px;
     display: inline-block;
-    padding: 5px;
+    padding: 14px 5px;
```

Comment:
Wouldn't be best here to have a padding of 15px instead of 14? to go from 5 to 5 and be able to reuse. What do you think?

Reply:
Yes, you're right